src/constants/actions.js

```diff
@@ -66,4 +66,6 @@ export const CLEAR_SHAPES = 'CLEAR_SHAPES'
 export const SAVE_EDITS = 'SAVE_EDITS'
 export const MUTATE_SHAPE = 'MUTATE_SHAPE'
 export const UNDO_EDIT = 'UNDO_EDIT'
-export const CLEAR_SHAPES_IN_PROGRESS = 'CLEAR_SHAPES_IN_PROGRESS'
```

Comment:
Is looks like `CLEAR_SHAPES_IN_PROGRESS` is still being used elsewhere in the code. Should it be removed there?

Reply:
Oops, not sure why this got deleted!